test/mocks/mocks.go

```diff
@@ -380,6 +382,7 @@ func ServiceBindingRequestMock(
 	applicationResourceRef string,
 	matchLabels map[string]string,
 	bindUnannotated bool,
+	groupVersionResource schema.GroupVersionResource,
```

Comment:
Typically this is called `gvr` in the rest of the project, I suggest the same here.